### PR TITLE
Add django-conf to encapsulate settings

### PR DESCRIPTION
**Pull request by khchine5**

### CODE REVIEW

File: pinax/comments/conf.py [hunk 1]
```diff
@@ -1,12 +1,9 @@
 from __future__ import unicode_literals
 
-
-from django.conf import settings
-
 from appconf import AppConf
+from django.conf import settings
```
Comment (paltman) [inline pinax/comments/conf.py]: @khchine5 
Can you add a `# noqa` like so:

```python
from django.conf import settings  # noqa
```

**Comment by paltman** [comment]
@khchine5 

Can you add `django-appconf` to the setup.py install requires?

```python
install_requires=[
        "django-appconf>=1.0.1"
]
```

**Comment by khchine5** [comment]
@paltman Done.

**Comment by paltman** [comment]
@khchine5 since the app label is pinax_comments, the settings names will be prefixed with PINAX_COMMENTS_. Also, in the app conf declaration you should just use the suffix (e.g. CAN_DELETE_CALLABLE).

**Comment by khchine5** [comment]
@paltman  I have fixed the issues and all tests pass now.

**Comment by paltman** [comment]
@khchine5 awesome sauce! thanks!

**Pull request merged**